server/irmaserver/api.go

```diff
@@ -145,7 +167,7 @@ func (s *Server) Stop() {
 		server.LogWarning(err)
 	}
 	s.stopScheduler <- true
-	s.sessions.stop()
+	_ = s.sessions.stop()
```

Comment:
Error will only be logged. I now removed the error response from the stop() function as it will never be used. That should improve the readability here.